shared/login/relogin/index.desktop.js

```diff
@@ -30,6 +30,8 @@ const UserRow = ({user}) => (
 )
 
 class Login extends React.Component<Props, State> {
+  _inputRef = React.createRef<Kb.Input>()
```

Comment:
need to explicitly type here because we build an object with `ref` instead of passing it directly to a component